docs/tutorials/embedding/AdjacencySpectralEmbed.ipynb

```diff
@@ -99,11 +141,103 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# fit and transform\n",
     "ase = AdjacencySpectralEmbed(n_components=2, algorithm='truncated')\n",
     "Xhat = ase.fit_transform(undirected_sbm)\n",
+    "\n",
+    "# plot\n",
     "pairplot(Xhat, title='2-component embedding', height=4)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Out-of-Sample (OOS) Embedding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Given new observations not seen in the original matrix, we sometimes wish to determine their latent positions without re-embedding the entire updated matrix.  \n",
+    "\n",
+    "More formally, suppose we have computed the embedding $\\hat{X} \\in \\textbf{R}^{n \\times d}$ from some adjacency matrix $A \\in \\textbf{R}^{n \\times n}$.  \n",
+    "Suppose we then obtain some new vertex $w \\in \\textbf{R}^n$ and we want to know its latent position without re-embedding the entire matrix.\n",
+    "\n",
+    "We can obtain an estimation for this out-of-sample latent position with ASE's `predict` method, for either single oos vertices or multiple."
```

Comment:
I didn't add a section on how this works with directed graphs, but I could if we want.

Reply:
normally i'd say no but the way the user has to do this is slightly different, so maybe worth showing?

Reply:
I'll add that in

Reply:
see 520a83b

Although, I'm planning on refactoring this into an oos-specific tutorial anyway, per our discussion in class on Monday.

Reply:
see b87a501 for oos-specific tutorial